tests/migrations/test_autodetector.py

```diff
@@ -862,11 +822,11 @@ def test_rename_m2m_through_model(self):
         Tests autodetection of renamed models that are used in M2M relations as
         through models.
         """
-        # Make state
-        before = self.make_project_state([self.author_with_m2m_through, self.publisher, self.contract])
-        after = self.make_project_state([self.author_with_renamed_m2m_through, self.publisher, self.contract_renamed])
-        autodetector = MigrationAutodetector(before, after, MigrationQuestioner({'ask_rename_model': True}))
-        changes = autodetector._detect_changes()
+        changes = self.get_changes(
+            [self.author_with_m2m_through, self.publisher, self.contract],
+            [self.author_with_renamed_m2m_through, self.publisher, self.contract_renamed],
+            MigrationQuestioner({"ask_rename_model": True})
```

Comment:
Should be changing double quotes to single, if at all. I wouldn't really worry about it though.